app/config/image_variations.yml

```diff
@@ -20,3 +20,9 @@ ezpublish:
                     reference: ~
                     filters:
                         - { name: geometry/scalewidthdownonly, params: [710] }
+liip_imagine:
+    driver: gd
+    filter_sets:
+        blog_thumbnail:
```

Comment:
I'm not sure if I understand you correctly. Do you want to be able to set conversion parameters globally and after that, these setting should be applied to every filter_set?